folium/folium.py

```diff
@@ -676,11 +663,9 @@ def _popup_render(self, popup=None, mk_name=None, count=None,
             Type of marker. Simple, Circle, etc.
         count: int, default None
             Count of marker
-        popup_on: boolean, default True
-            If False, no popup will be rendered
         """
-        if not popup_on:
-            return 'var no_pop = null;'
+        if not popup:
+            return ''
```

Comment:
Ah, so setting it to `null` does not change that.  Let me take a closer look at this later (I am still travelling...)

If that is what leaflet does then lets merge as is.